stingray/crossspectrum.py

```diff
@@ -287,6 +302,23 @@ def rebin(self, df, method="mean"):
         bin_cs.nphots1 = self.nphots1
         bin_cs.power_err = binerr
 
+        if hasattr(self, 'unnorm_power'):
+            _, binpower_unnorm, _, _ = \
+                rebin_data(self.freq, self.unnorm_power, df,
+                           method=method, dx=self.df)
+
+            bin_cs.unnorm_power = binpower_unnorm
+
+        if hasattr(self, 'cs_all'):
+            cs_all = []
+            for c in self.cs_all:
+                cs_all.append(c.rebin(df=df, f=f, method=method))
+            bin_cs.cs_all = cs_all
+        if hasattr(self, 'pds1'):
+            bin_cs.pds1 = self.pds1.rebin(df=df, f=f, method=method)
+        if hasattr(self, 'pds2'):
+            bin_cs.pds2 = self.pds2.rebin(df=df, f=f, method=method)
```

Comment:
Not if none is true, in general they will both be true for cross spectra, unless the main object is actually a power spectrum